packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -450,39 +520,6 @@ contract ChugSplashManager is Owned {
         registry.announce("ChugSplashBundleClaimed");
     }
 
-    /**
-     * @notice Refunds `executorBondAmount` to the executor if they complete the bundle within
-     *        `executionLockTime`, or if the project owner cancels the bundle.
-     *
-     * @param _bundleId ID of the bundle that was completed or cancelled by the project owner.
-     */
-    function returnExecutorBond(bytes32 _bundleId) external {
```

Comment:
Changed this into an internal function since it's only called by the ChugSplashManager

Reply:
I think we should do a pull-based model instead of a push-based model, so I did like having this be public. Did you see issues with having it public?